src/main/java/org/sunbird/user/service/UserUtilityServiceImpl.java

```diff
@@ -589,9 +589,11 @@ public String createBulkUploadUser(UserRegistration userRegistration) {
 		requestBody.put(Constants.PHONE_VERIFIED, true);
 		requestBody.put(Constants.ROLES, Arrays.asList(Constants.PUBLIC));
 		request.put(Constants.REQUEST, requestBody);
+		Map<String, String> headerValues = ProjectUtil.getDefaultHeaders();
+		headerValues.put(Constants.X_AUTH_TOKEN, userRegistration.getUserAuthToken());
```

Comment:
Need to add this header only if the value is available.
getDefaultHeaders adds content-type header -- is this added again ?

Reply:
added the empty check